scripts/storage-mount.sh

```
@@ -0,0 +1,258 @@
+# handful of keys that actually appear on the right hand side.
+expand_env() {
+  local v="$1"
+  v="${v//\$\{DATA_FOLDER\}/$DATA_FOLDER}"
+  v="${v//\$\{CONFIG_FOLDER\}/$CONFIG_FOLDER}"
+  printf '%s' "$v"
+}
+
+DATA_FOLDER="$(env_value DATA_FOLDER || printf './data')"
+CONFIG_FOLDER="$(env_value CONFIG_FOLDER || printf './configs')"
+STORAGE_REMOTE="$(env_value STORAGE_REMOTE || printf '')"
+STORAGE_MOUNTPOINT="$(expand_env "$(env_value STORAGE_MOUNTPOINT || printf "$DATA_FOLDER")")"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)